src/ProjectService.Data/UserRepository.cs

```diff
@@ -95,5 +95,16 @@ public List<DbProjectUser> Find(List<Guid> userIds)
         {
             return _provider.ProjectsUsers.Where(u => userIds.Contains(u.UserId)).ToList();
         }
+
+        public void Remove(Guid userId)
```

Comment:
Why consumer named diactivate but repository method named as remove?

Reply:
in other repositories methods like this named "remove"